Examples/accepting-values-test.lisp

```diff
@@ -61,7 +61,10 @@
     (present '(com-accept-popup) 'command :stream pane)
     (fresh-line pane)
     (present '(com-accepting-with-list-pane-view) 'command :stream pane)
-    (fresh-line pane)))
+    (fresh-line pane)
+    (present '(com-accepting-with-gadgets) 'command :stream pane)
+    (fresh-line pane)
+    ))
```

Comment:
Please put the closing parens on the previous line.

Reply:
Ok, fixed.